### PR TITLE
Archive changes

### PR DESCRIPTION
**Pull request by tymees**
This PR addresses #369 and #404

I've also renamed to semi-public archive to private archive in the code to avoid confusion. I wouldn't say it's a perfect name, but at least it's more distinct that semi-public. 

I've left in the site-export functionality (the mechanism for filling the current page on the WP). Just for the transition period.

### CODE REVIEW

**Review by Meesch — COMMENTED** [review]
I somehow can't get the list to work on my machine... I'm sure I'm missing something. I have this approved and closed submission, for example:
Screenshot 2022-11-08 at 12 26 37

But it won't show up on the archive page:
Screenshot 2022-11-08 at 12 27 45

File: locale/en/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -2798,6 +2812,39 @@ msgstr ""
...
 msgstr "Submit application for preliminary assessment"
 
+#: proposals/templates/proposals/proposal_public_archive.html:8
+#: proposals/templates/proposals/proposal_public_archive.html:14
+msgid "Goedgekeurde aanvragen"
+msgstr "Approved applications;"
+
+#: proposals/templates/proposals/proposal_public_archive.html:18
+msgid "Referentie nummer"
+msgstr "Reference Number"
```
[the remark this answers is not in the document]
Comment (tymees) [reply locale/en/LC_MESSAGES/django.po:2822]: I'm confused? In this PR everything I added should be capitalized? (Both Dutch and English)

The page on the WP is indeed all over the place, dunno why

File: locale/en/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -2798,6 +2812,39 @@ msgstr ""
...
+msgid "Aanvrager"
+msgstr "Applicant"
+
+#: proposals/templates/proposals/proposal_public_archive.html:22
+msgid "Datum goedgekeurd"
+msgstr "Approval date"
+
+#: proposals/templates/proposals/proposal_public_archive.html:23
+msgid "FETC-GW Kamer"
+msgstr "FEtC-H cahnger"
```
[the remark this answers is not in the document]
Comment (tymees) [reply locale/en/LC_MESSAGES/django.po:2838]: Derp, fixed

File: locale/en/LC_MESSAGES/django.po (hunk 1 above)
[the remark this answers is not in the document]
Comment (tymees) [reply locale/en/LC_MESSAGES/django.po]: Ooh wait I see it now... I've made it more consistent

**Comment by tymees** [comment]
> But it won't show up on the archive page:

I had this as well, it's a caching issue. Try opening the API endpoint in your browser and do a force refresh

**Pull request merged**